tls/ttls.h

```diff
@@ -560,6 +562,7 @@ typedef struct ttls_context {
 	unsigned int		nb_zero;
 	int			client_auth;
 	char			*hostname;
+	TfwVhost		*vhost;
```

Comment:
Please use `void *` here

Reply:
That's one more reason reason to compare SNI with the request authority as string: semantically this thing should not be an opaque pointer.